XenAdmin/Wizards/PatchingWizard/PatchingWizard_AutomatedUpdatesPage.resx

```diff
@@ -271,19 +271,79 @@
     <value>True</value>
   </data>
   <data name="panel1.ColumnCount" type="System.Int32, mscorlib">
+    <value>3</value>
+  </data>
+  <data name="buttonRetry.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="buttonRetry.Location" type="System.Drawing.Point, System.Drawing">
+    <value>457, 3</value>
+  </data>
+  <data name="buttonRetry.Size" type="System.Drawing.Size, System.Drawing">
+    <value>51, 23</value>

```

Comment:
I think the Retry button should be the same size as the 3 wizard progress buttons at the bottom - I know it takes up more size, but as it is shorter, it looks a bit strange.